lib/lacerda/tasks.rb

```diff
@@ -5,6 +5,7 @@ class Tasks
     include Rake::DSL if defined? Rake::DSL
 
     def install_tasks
+      Bundler.require
```

Comment:
This is here because otherwise `lib/lacerda/drafter` is loaded, and it fails because the [ffi_lib file](https://github.com/moviepilot/lacerda/pull/30/files#diff-2ef2b71b4811806413dbf57f28c2e627R28) is not yet compiled